src/lib/sidenav/drawer.ts

```diff
@@ -153,6 +155,9 @@ export class MdDrawer implements AfterContentInit, OnDestroy {
   /** Event emitted when the drawer's position changes. */
   @Output('positionChanged') onPositionChanged = new EventEmitter<void>();
 
+  /** Event emitted when the drawer's mode changes. */
+  @Output('modeChanged') onModeChanged = new EventEmitter<void>();
```

Comment:
since this is really just for internal coordination, can we make it a `Subject` with no `@Output()` and `_` name?

Reply:
Sure, I did this to keep it consistent with the `positionChanged`.